content/en/docs/instrumentation/java/exporters.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Exporters
+weight: 50
+---
+
+In order to visualize and analyze your traces, you will need to export them to a
+backend such as [Jaeger](https://www.jaegertracing.io/) or
+[Zipkin](https://zipkin.io/). OpenTelemetry Java provides exporters for some
+common open source backends.
+
+If you use the Javaagent for
+[automatic instrumentation](/docs/instrumentation/java/automatic) you can learn
+how to setup exporters following the
+[Agent Configuration Guide](/docs/instrumentation/java/automatic/agent-config)
+
+For [manual instrumentation](/docs/instrumentation/java/manual), you will find
+some introductions below on how to set up backends and the matching exporters.
+
+## OTLP endpoint
+
+To send trace data to a OTLP endpoint (like the [collector](/docs/collector) or
+Jaeger) you'll want to use `opentelemetry-exporter-otlp`:
+
+<!-- prettier-ignore-start -->
```

Comment:
Just curious why you added `prettier-ignore` directives? They shouldn't be needed anymore.

Reply:
probably a copy&paste error or I forgot to remove them